### DISCUSSION TITLE
Documentation is missing for clr-icon

### DISCUSSION THREAD

**Discussion by dazipe** [vmware-clarity/ng-clarity#513] [General]
Hey everybody,

I realize the need to switch to the cds-icon from clr-icon, but what I don't understand is why the documentation was removed?
https://clarity.design/icons
We still use the clr-icon in our project and I guess many others do to. 
Please restore the documentation page.

**Comment by kevinbuhmann** [comment]
The @clr/icons docs can be found here: https://v12.clarity.design/icons.

**Reply by dazipe** [reply]
This is great! Thank you very much.
But I could not find in with Google. Be nice though if either the https://clarity.design/icons page would be restored or at least it would include link to the https://v12.clarity.design/icons

**Reply by kevinbuhmann** [reply]
I can add a link there. The link is currently just on the Clarity Core icons page, so you have to click two links to find it. Sorry about that.
